qubes-rpc/qubes.GetDate

```diff
@@ -16,7 +16,10 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-#
-#
 
-date -u -Iseconds
+export LC_ALL=C
+if [ -n "${1-}" ] && [ "$1" -ge 1 ]; then
```

Comment:
Maybe make the argument a bit more descriptive? Like "nanoseconds", or "ns"?

Reply:
The argument was intended to be a version number, in case we decide to change this again in the future.  I can use a more descriptive string instead, though.